save-backend/src/main/kotlin/com/saveourtool/save/backend/storage/TestsSourceSnapshotStorage.kt

```diff
@@ -25,39 +24,47 @@ import kotlin.io.path.*
 class TestsSourceSnapshotStorage(
     s3Operations: S3Operations,
     s3KeyManager: TestsSourceSnapshotS3KeyManager,
+    private val configProperties: ConfigProperties,
 ) : ReactiveStorageWithDatabase<TestsSourceSnapshotDto, TestsSourceSnapshot, TestsSourceSnapshotS3KeyManager>(
     s3Operations,
     s3KeyManager,
 ) {
+    private fun createTempDirectoryForArchive() = Files.createTempDirectory(
+        configProperties.workingDir,
+        "source-"
+    )
+
     /**
      * @param request
      * @return [TestFilesContent] filled with test files
      */
-    fun getTestContent(request: TestFilesRequest): Mono<TestFilesContent> {
-        val tmpSourceDir = createTempDirectory("source-")
-        val tmpArchive = createTempFile(tmpSourceDir, "archive-", ARCHIVE_EXTENSION)
-        val sourceContent = download(request.testsSourceSnapshot)
-            .map { DefaultDataBufferFactory.sharedInstance.wrap(it) }
```

Comment:
dont quite understand where did `DefaultDataBufferFactory` and `DataBufferUtils` disappear to

Reply:
re-use existed method `collectToFile` which uses `DefaultDataBufferFactory` inside